### PR TITLE
Creating the ad component

### PR DESCRIPTION
**Pull request by MoisesFlores22**
What this PR does?

Create a generic component to add an image or ad to the top

Pictures:

When an image is load the chat would looks like this 

screen shot 2017-02-08 at 4 29 29 pm 2
 

if there is no image will show something like this

screen shot 2017-02-08 at 4 37 10 pm 2

### CODE REVIEW

File: src/Ad.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
```
Comment (erikvilla) [inline src/Ad.tsx]: I don't think you need the whole library, it is a best practice to just import what you need

File: src/Ad.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  constructor(props: Props) {
+      super(props);
+  }
+
+  render () {
+    const { img_src, link } = this.props;
+    if (img_src) {
+      return(
+        <div className='wc-add'>
+          <a href={link} target="_blank"><img src={img_src} /></a>
```
Comment (erikvilla) [inline src/Ad.tsx]: There is duplicated code here, you can put the dynamic part of it in a variable and just include it

File: static/assets/We-know-HOU-white-black-back .svg [hunk 3]
```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (erikvilla) [inline static/assets/We-know-HOU-white-black-back .svg]: There is a space in the file name

Comment (MoisesFlores22) [inline static/assets/We-know-HOU-white-black-back .svg]: yeah chron pass it like this, fixing it

File: src/Ad.tsx (hunk 1 above)
Comment (Alex-TH) [inline src/Ad.tsx]: For example you can only import Component and in the class declaration extends directly from Component without the need to put React.Component

Comment (MoisesFlores22) [inline src/Ad.tsx]: that form throws me an error 😞

**Comment by MoisesFlores22** [comment]
can I merge or should i wait for more reviews?

**Comment by MoisesFlores22** [comment]
@AlexisGlez check it

File: src/Ad.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+  constructor(props: Props) {
+      super(props);
+  }
+
+  render () {
+    const { img_src, link } = this.props;
+    const image = img_src ? <img src={img_src} /> : null;
+    return(
+      <div className='wc-add'>
```
Comment (AlexisGlez) [inline src/Ad.tsx]: @MoisesFlores22 I think you can change the className `wc-add` by `wc-ad`

File: botchat.css [hunk 5]
```diff
@@ -24,6 +24,26 @@
...
+      z-index: 1;
+      line-height: 1.5;
+      background-color: black;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      padding: 0 10px;
+    }
+
+    .wc-add img {
```
Comment (AlexisGlez) [inline botchat.css]: wc-add -> wc-ad

File: botchat.css [hunk 6]
```diff
@@ -24,6 +24,26 @@
         display: block;
     }
 
+    .wc-add {
```
Comment (AlexisGlez) [inline botchat.css]: wc-add -> wc-ad

**Pull request merged**